changelog/may2025/2025-05-14-instances-removed-block-storage-volume-creation-removed.mdx

```diff
@@ -0,0 +1,10 @@
+---
+title: Block Storage volume creation removed from the Instance API
+status: removed
+date: 2025-05-01
+category: compute
+product: instances
+---
+
+Since April 30th 2025, it has become impossible to **CREATE** new block volumes with the Instance API (or through associated devtools). It means that customers will only be in capacity to list, mount, unmount or delete volumes through the Instance API. Block volume creation must go through the Block Storage API. [Learn more about the advantages of migrating from the Instance API to the Block Storage API for managing block volumes and snapshots](https://www.scaleway.com/en/docs/block-storage/reference-content/advantages-migrating-to-sbs/).
```

Comment:
```suggestion
Since April 30th 2025, it has become impossible to **CREATE** new block volumes with the Instance API (or through associated devtools). It means that customers will only be in capacity to list, mount, unmount or delete volumes through the Instance API. Block volume creation must go through the Block Storage API. [Learn more about the advantages of migrating from the Instance API to the Block Storage API for managing block volumes and snapshots](/block-storage/reference-content/advantages-migrating-to-sbs/).
```